examples/garrawarla/configs/packages.yaml

```diff
@@ -0,0 +1,193 @@
+packages:
+  all:
+    target: [broadwell]
+    compiler: [gcc@8.3.0, gcc@4.8.5]
+    providers:
+      mpi: [openmpi]
+      blas: [intel-mkl]
+      lapack: [intel-mkl]
+      scalapack: [intel-mkl]
+      fftw-api: [intel-mkl]
+      mkl: [intel-mkl]
+    permissions:
+      read: world
+      write: group
```

Comment:
`write: user` probably better, it will restrict to maali/spack (whichever will be)